README.md

```diff
@@ -1,3 +1,7 @@
+<p align="center">
```

Comment:
Nit: `align` attribute on `p` [is obsolete](https://dev.w3.org/html5/spec-LC/obsolete.html).

```suggestion
<p style="text-align: center;">
```

Reply:
This doesn't seem to work correctly:
![image](https://github.com/Azure/trusted-signing-action/assets/59936622/c1900987-b67d-4256-930c-84150c64c5d5)

Reply:
Oops, GitHub doesn't appear to support this. Ignore.

Reply:
No problem. Thank you for the review!